packages/react/src/components/Storage/FileUploader/Previewer/index.tsx

```diff
@@ -1,6 +1,67 @@
 import React from 'react';
-import { Card } from '../../../../primitives';
+import { translate } from '@aws-amplify/ui';
+import { PreviewerProps } from '../types';
+import { Button, Card, Flex, Text, View } from '../../../../primitives';
+import { UploadDropZone } from '../UploadDropZone';
+import { UploadButton } from '../UploadButton';
+import { Tracker } from '../Tracker';
 
-export function Previewer(): JSX.Element {
-  return <Card>Previewer</Card>;
+export function Previewer({
+  acceptedFileTypes,
+  files,
+  inDropZone,
+  multiple,
+  onClose,
+  onDragEnter,
+  onDragLeave,
+  onDragOver,
+  onDragStart,
+  onDrop,
+  onFileChange,
+}: PreviewerProps): JSX.Element {
+  const CommonProps = {
+    acceptedFileTypes,
+    multiple,
+    onFileChange,
+  };
```

Comment:
Interesting pattern. You don't have to specify all the props, you can use props destructuring and then pass that to sub-components instead of this CommnonProps. (see example [here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/src/primitives/Button/Button.tsx#L22))

Reply:
OK I'll try that. Probably don't need `CommonProps` but I was copying a pattern I used in a different component.